tests/OSPSuite.Core.Tests/Core/SimModelExporterIntegrationTests.cs

```diff
@@ -40,9 +39,8 @@ protected override void Context()
          _objectPathFactory = IoC.Resolve<IObjectPathFactory>();
          var solverSettingsFactory = IoC.Resolve<ISolverSettingsFactory>();
          _solverSettings = solverSettingsFactory.CreateCVODE();
-         _tableFormulaToTableFormulaExportMapper = IoC.Resolve<ITableFormulaToTableFormulaExportMapper>();
-         _concentrationBasedFormulaUpdater = IoC.Resolve<IConcentrationBasedFormulaUpdater>();
-         sut = new SimModelExporter(new CreateExportModelVisitor(_objectPathFactory, _tableFormulaToTableFormulaExportMapper,_concentrationBasedFormulaUpdater), new ExportSerializer(_simModelXmlSerializerRepository));
+         _simulationExporterCreatorFactory = IoC.Resolve<ISimulationExportCreatorFactory>();
+         sut = new SimModelExporter(_simulationExporterCreatorFactory, new ExportSerializer(_simModelXmlSerializerRepository));
```

Comment:
I am not sure why the object is created like that but I don't have time to investigate all the weirdness as long as it works